### ISSUE TITLE
train and loss curve

### ISSUE THREAD

**Issue by liaojiacai** [ProbabilisticNumerics/cabs#1]
hi，i am very glad to see you have provided the optimizer code,well, i want to compare the optimizer with my own method, so can you provide the train and test result data? if you can provide those result data in your paper, i will solve a lot time,beause all my code were wtitten by pytorch,it will save me a lot of time,Thanks in advance!!!

**Comment by lballes** [comment]
Hi, and thanks for your interest! Can you e-mail me about this at lukas dot balles at tuebingen dot mpg dot de?

**Comment by liaojiacai** [comment]
my e-mail is：943906785@qq.com，thanks you very much！！！------------------ 原始邮件 ------------------
发件人: "Lukas Balles"<notifications@github.com>
发送时间: 2019年6月22日(星期六) 凌晨0:19
收件人: "ProbabilisticNumerics/cabs"<cabs@noreply.github.com>;
抄送: "Opencv-and-Opengl"<943906785@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [ProbabilisticNumerics/cabs] train and loss curve (#1)



Hi, and thanks for your interest! Can you e-mail me about this at lukas dot balles at tuebingen dot mpg dot de?
 
&mdash;

**Comment by liaojiacai** [comment]
well，i will reply as soon as possible！------------------ 原始邮件 ------------------
发件人: "Lukas Balles"<notifications@github.com>
发送时间: 2019年6月22日(星期六) 凌晨0:19
收件人: "ProbabilisticNumerics/cabs"<cabs@noreply.github.com>;
抄送: "Opencv-and-Opengl"<943906785@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [ProbabilisticNumerics/cabs] train and loss curve (#1)



Hi, and thanks for your interest! Can you e-mail me about this at lukas dot balles at tuebingen dot mpg dot de?
 
&mdash;
